install/v2plugin/README.md

```diff
@@ -16,30 +16,54 @@ Contiv plugin config options should be specified if it is different from default
 
 docker plugin install contiv/v2plugin:<version-tag> ARG1=VALUE1 ARG2=VALUE2 ...
 ```
-ARG           : DESCRIPTION                                   : DEFAULT VALUE
---------------:-----------------------------------------------:----------------------
-iflist        : VLAN uplink interface used by OVS             : ""
-cluster_store : Etcd or Consul cluster store url              : etcd://localhost:2379
-ctrl_ip       : Local IP address to be used by netplugin      : none
-                for control communication
-vtep_ip       : Local VTEP IP address to be used by netplugin : none
-plugin_role   : In 'master' role, plugin runs netmaster       : master
-                and netplugin
-listen_url    : Netmaster url to listen http requests on      : ":9999"
-control_url   : Netmaster url for control messages            : ":9999"
-dbg_flag      : To enable debug mode, set to '-debug'         : ""
-fwd_mode      : Forwarding mode                               : ""
-
+ARG                               : DESCRIPTION                                                               : DEFAULT VALUE
+----------------------------------:---------------------------------------------------------------------------:----------------------
+CONTIV_ROLE                       : contiv net service net, options: [netmaster, netplugin]                   : ""
+CONTIV_LOG_DIR                    : contiv log file directory                                                 : "/var/log/contiv"
+CONTIV_NETPLUGIN_CONSUL_ENDPOINTS : a comma-delimited list of netplugin consul endpoints                      : ""
+CONTIV_NETPLUGIN_ETCD_ENDPOINTS   : a comma-delimited list of netplugin etcd endpoints                        : ""
```

Comment:
that introduce extra layer in the startcontiv.sh and i'm trying to avoid it ...
we can default to use etcd `http://127.0.0.1:2379` if one of them are set, but it needs to be in `configs.go` instead of startcontiv.sh. thought ?

Reply:
if you meant "none" instead of "one" . . seems good, we can still document the default as etcd even if it's not done in startcontiv?